src/guessNumber.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const { isNumberValid } = require('./isNumberValid');
+const { findBullsAndCows } = require('./findBullsAndCows');
+
+const terminal = require('./terminal').terminal;
+
+const messages = {
+  error: 'Your number is not correct, try to enter another one\n',
+  tryAgainMessage: 'Please, try again\n',
+  success: 'You did it, Game over!)',
+};
+
+const guessNumber = (secret, message) => {
+  terminal.question(message, (guess) => {
+    if (!isNumberValid(guess)) {
+      guessNumber(secret, messages.error);
+
+      return;
+    }
+
+    const { bulls, cows } = findBullsAndCows(secret, guess);
+
+    if (bulls !== 4) {
```

Comment:
What is this number actually mean?
Do you remember why it's equal to 4 after a month?
It would help if you created a specific variable for such cases